server.go

```diff
@@ -19,6 +19,9 @@ import (
 const (
 	// SftpServerWorkerCount defines the number of workers for the SFTP server
 	SftpServerWorkerCount = 8
+
+	defaultFileMode = 0o644
+	defaultDirMode  = 0o755
```

Comment:
These constants should be grouped individually in their own `const` so that a block-level documentation can cover them both.

Prefer to also have them be typed constants, so that we can simply use `mode := defaultDirMode` .